### DISCUSSION TITLE
Customization of Manifest File Directory in Vite Plugin

### DISCUSSION THREAD

**Discussion by tobigumo** [remix-run/remix#8784] [General]
I am currently using the Vite plugin and have encountered a situation where I needed to modify the client assets directory from the default assets directory. To achieve this, I adjusted the rollup settings in my vite.config.ts file as shown below:

```ts:vite.config.ts
export default defineConfig({
  // ... 
  build: {
    assetsDir: "other-assetdir",
  }
});
```

Despite this configuration change, the manifest-[hash].js file continues to be generated in the assets directory. I was exploring the Vite plugin codebase and noticed that the current behavior is defined in the Remix repository:
https://github.com/remix-run/remix/blob/f8cdb3093e9b598fe074021a41da27d8a3e31c9c/packages/remix-dev/vite/plugin.ts#L779

I would like to request an enhancement that allows users to customize the directory for the manifest-[hash].js file through the Vite plugin settings. This flexibility would be greatly beneficial for users who, like me, have specific requirements for organizing their project assets.

I look forward to the vite becoming stable supported!

**Comment by chrisvariety** [comment]
This was resolved in https://github.com/remix-run/remix/pull/8843 ✨

**Comment by tobigumo** [comment]
Great! I'm glad to hear that this issue has been resolved.
Thx!
